deploy/helm/sumologic/values.yaml

```diff
@@ -4155,461 +4155,23 @@ metadata:
-          ## This is the same pipeline like logs/fluent/systemd, but with the following changes:
-          ## - otlp receiver instead of fluentforward
-          ## - added transform/remove_attributes processor
-          logs/otlp/systemd:
-            receivers:
-              - otlp
-            processors:
-              - memory_limiter
-              - filter/include_fluent_tag_host
-              - filter/include_systemd
-              - filter/exclude_kubelet
-              - filter/exclude_systemd_syslog
-              - filter/exclude_systemd_hostname
-              - filter/exclude_systemd_priority
-              - filter/exclude_systemd_unit
-              - attributes/extract_systemd_source_fields
-              - attributes/remove_fluent_tag
-              - groupbyattrs/systemd
-              - resource/add_cluster
-              - source/systemd
-              - transform/remove_attributes
-              - batch
-            exporters:
-              - sumologic/systemd
-          ## This is the same pipeline like logs/fluent/kubelet, but with the following changes:
-          ## - otlp receiver instead of fluentforward
-          ## - added transform/remove_attributes processor
-          logs/otlp/kubelet:
-            receivers:
-              - otlp
-            processors:
-              - memory_limiter
-              - filter/include_fluent_tag_host
-              - filter/include_kubelet
-              - filter/exclude_kubelet_syslog
-              - filter/exclude_kubelet_hostname
-              - filter/exclude_kubelet_priority
-              - filter/exclude_kubelet_unit
-              - attributes/extract_systemd_source_fields
-              - attributes/remove_fluent_tag
-              - groupbyattrs/systemd
-              - resource/add_cluster
-              - source/kubelet
-              - transform/remove_attributes
-              - batch
-            exporters:
-              - sumologic/systemd
+      ## Directly alter the OT configuration. The value of this key should be a dictionary, that will
+      ## be directly merged with the generated configuration, overriding existing values.
+      ## For example:
+      # override:
+      #   processors:
+      #     batch:
+      #       send_batch_size: 512
+      ## will change the batch size of the pipeline.
+      ##
+      ## WARNING: This field is not subject to backwards-compatibility guarantees offered by the rest
+      ## of this chart. It involves implementation details that may change even in minor versions.
+      ## Use with caution, and consider opening an issue, so your customization can be added in a safer way.
+      merge: {}
+      ## Completely override existing config and replace it with the contents of this value.
+      ## The value of this key should be a dictionary, that will replace the normal configuration.
+      ## This is an advanced feature, use with caution, and review the generated configuration first.
+      override: {}
```

Comment:
Please add information about new parameters and remove information about `otelcol.config` in https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md

Reply:
here: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/fluentd-otc-comparison.md also information about metadata.logs.config should be changed

Reply:
here: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/best-practices.md#collecting-logs-from-varlogpods I also found the information about `metadata.logs.config`

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/vagrant/values.yaml#L180-L182 should be also adjusted

Reply:
I found also this config: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/7b80b8aad023d633c6833242c1ec4b762e12788b/tests/integration/values/values_helm_default_otelcol_logs.yaml#L19-L21